src/go/configinfo/service_info.go

```diff
@@ -504,6 +510,22 @@ func (s *ServiceInfo) processUsageRule() error {
 	return nil
 }
 
+func (s *ServiceInfo) processJwtInQueryParams() {
+	authn := s.serviceConfig.GetAuthentication()
+	for _, provider := range authn.GetProviders() {
+		for _, jwtLocation := range provider.JwtLocations {
+			switch jwtLocation.In.(type) {
+			case *confpb.JwtLocation_Query:
+				s.JwtQueryParams[jwtLocation.GetQuery()] = true
+			default:
+				continue
+			}
+		}
+	}
+
+	s.JwtQueryParams[util.DefaultJwtQueryParamAccessToken] = true
```

Comment:
I thought that if a user defines a provider, then we will not use the default anymore?

Reply:
Theoretically, they can define the combination of custom ones and default ones.